CHANGELOG.md

```
@@ -1,5 +1,36 @@
 # Changelog
 
+## [Unreleased]
+
+## [0.1.3] - 2026-04-29
+
+### Fixed
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)